cnv/cnv_users_guide/cnv-dashboard.adoc

```diff
@@ -0,0 +1,13 @@
+[id="cnv-dashboard"]
+= Using the CNV Dashboard to get cluster information
```

Comment:
We're not supposed to abbreviate it to CNV. I would just call it the (cluster?) Overview Dashboard, if that's its official name. I'm also not sure if Overview Dashboard or Cluster Overview Dashboard should be capitalized or not.

Reply:
Thanks, changed to "overview dashboard" that describes cluster information (no CNV references).